Card/build.gradle

```diff
@@ -41,12 +41,12 @@ android {
 dependencies {
     api project(':Core')
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.6.0'
-    implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation 'com.google.android.material:material:1.4.0'
-    testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    implementation deps.kotlinStdLib
+    implementation deps.androidxCoreKtx
+    implementation deps.androidxAppcompat
+    implementation deps.material
```

Comment:
Do we need material outside of the demo app?

Reply:
Good catch. I don't think we do at the moment. I'll remove material from the card and core modules.